isthmus/src/test/java/io/substrait/isthmus/ComparisonFunctionTest.java

```diff
@@ -0,0 +1,29 @@
+package io.substrait.isthmus;
+
+import java.io.*;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class ComparisonFunctionTest extends PlanTestBase {
+
+  static List<String> CREATES =
+      List.of("CREATE TABLE numbers (c1 TINYINT, c2 TINYINT, c3 TINYINT)");
+
+  @Test
+  void between() throws Exception {
+    String query = "SELECT c1 FROM numbers WHERE c1 BETWEEN 1 AND 100";
+    assertSqlSubstraitRelRoundTrip(query, CREATES);
+  }
```

Comment:
This test doesn't actually exercise `between` because during query parsing, Calcite converts BETWEEN into an equivalent using two comparisons and an AND.
```
LogicalProject(C1=[$0])
  LogicalFilter(condition=[AND(>=($0, 1), <=($0, 100))])
    LogicalTableScan(table=[[NUMBERS]])
```